Skyline.DataMiner.Sdk - Support CatalogReferences (#4357)
SkylineCommunications/dataminer-docs: develop/CICD/Skyline DataMiner Software Development Kit/skyline_dataminer_sdk.md

---
uid: skyline_dataminer_sdk

> keywords: Skyline.DataMiner.Sdk, Software Development Kit

Reviewer: ```suggestion
keywords: Skyline.DataMiner.Sdk, Software Development Kit, Package, Catalog, dmapp, cicd
```

- **\*.dmapp file**: Represents a DataMiner application package containing the content of your project, ready to be installed on a DataMiner System.
- **\*.zip file**: Represents Catalog information used for publishing the package to the DataMiner Catalog.

> Only applicable for the **DataMiner Package Project**, it also checks the `CatalogDefaultDownloadKeyName` property for the name of an environment variable or Visual Studio secret containing a [organization key](xref:Managing_DCP_keys#organization-keys) to download Catalog items defined in the CatalogReferences.xml.

Reviewer 2: Is this what you mean here?
```suggestion
Only for the **DataMiner Package Project**, the SDK also checks the `CatalogDefaultDownloadKeyName` property for the name of an environment variable or Visual Studio secret containing an [organization key](xref:Managing_DCP_keys#organization-keys) to download Catalog items defined in the CatalogReferences.xml.
```
